pkg/virt-launcher/virtwrap/access-credentials/access_credentials_test.go

```diff
@@ -368,9 +363,8 @@ var _ = Describe("AccessCredentials", func() {
 		manager.stopCh = make(chan struct{})
 		password = password + "morefake"
 		Expect(os.WriteFile(filepath.Join(secretDirs[0], user), []byte(password), 0644)).To(Succeed())
-		base64Str = base64.StdEncoding.EncodeToString([]byte(password))
-		cmdSetPassword = fmt.Sprintf(`{"execute":"guest-set-user-password", "arguments": {"username":"%s", "password": "%s", "crypted": false }}`, user, base64Str)
-		mockConn.EXPECT().QemuAgentCommand(cmdSetPassword, domName).MinTimes(1).Return("", nil)
+
+		mockDomain.EXPECT().SetUserPassword(user, password, libvirt.DomainSetUserPasswordFlags(0)).MinTimes(1).Return(nil)
```

Comment:
Same

Reply:
Same thing as I replied above, it's called multiple times. We've discussed about it together with @akrejcir privately.